code/mob/living.dm

```diff
@@ -769,6 +769,7 @@
 	var/italics = 0
 	var/forced_language = null
 	var/message_range = null
+	var/skip_maptext = null
```

Comment:
why null, why not 0 like the other one, or even better - true and false

Reply:
was attempting to be consistent across with other ones- which was silly hdsjksd
fixed now!